src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -44,7 +44,7 @@ class JesBackendCall(val backend: JesBackend,
   import JesBackend._
   import JesBackendCall._
 
-  def jesCommandLine = s"/bin/bash ${cmdInput.local}"
+  def jesCommandLine = s"/bin/bash ${JesWorkingDisk.MountPoint}/${cmdInput.local}"
```

Comment:
Yeah that thought crossed my mind when I was initially writing this... I'll go through and fix these.